tests/testthat/test-window.R

```diff
@@ -14,35 +14,39 @@ rec <- recipe(~., data = sim_dat)
 
 test_that("error checks", {
   expect_snapshot(error = TRUE,
-    rec %>% step_window(y1, size = 6)
+    rec %>% step_window(y1, size = 6) %>% prep()
   )
   expect_snapshot(error = TRUE,
-    rec %>% step_window(y1, size = NA)
+    rec %>% step_window(y1, size = NA) %>% prep()
   )
   expect_snapshot(
     error = TRUE,
-    rec %>% step_window(y1, size = NULL)
+    rec %>% step_window(y1, size = NULL) %>% prep()
   )
   expect_snapshot(error = TRUE,
     rec %>% step_window(y1, statistic = "average")
   )
   expect_snapshot(error = TRUE,
-    rec %>% step_window(y1, size = 1)
+    rec %>% step_window(y1, size = 1) %>% prep()
   )
   expect_snapshot(error = TRUE,
-    rec %>% step_window(y1, size = 2)
+    rec %>% step_window(y1, size = 2) %>% prep()
   )
   expect_snapshot(error = TRUE,
-    rec %>% step_window(y1, size = -1)
+    rec %>% step_window(y1, size = -1) %>% prep()
   )
   expect_snapshot(
-    rec %>% step_window(y1, size = pi)
+    rec %>% step_window(y1, size = 3 + .Machine$double.eps) %>% prep()
```

Comment:
So I don't know that we need both of the tests using `.Machine$double.eps` but it did affirm by beleif of what _should_ happen given how `all_equal()` works.